packages/lesswrong/lib/types/collectionTypes.ts

```diff
@@ -335,6 +335,7 @@ interface ResolverContext extends CollectionsByName {
   locale: string,
   isSSR: boolean,
   isGreaterWrong: boolean,
+  isIssaRiceReader: boolean,
```

Comment:
should be optional or should be included in the other functions which create a context (i.e. `createAnonymousContext` and `createAdminContext`) to prevent type errors elsewhere